opentelemetry/sidecar/README.md

```diff
@@ -0,0 +1,40 @@
+# Deployment
```

Comment:
Is this title intentional or is it copy-paste from the deployment example?

Reply:
https://github.com/os-observability/redhat-rhosdt-samples/tree/main/opentelemetry/deployment#deployment

Reply:
An error from the copy/paste